modules/ROOT/partials/configuration/end_container_on_empty_block.adoc

```diff
@@ -1,20 +1,20 @@
 [[end_container_on_empty_block]]
 == `+end_container_on_empty_block+`
 
-This option allows you to to split the current container block element if the enter key is pressed inside an empty inner block element.
+This option allows you to split the current container block element and specify the type of element created if the enter key is pressed inside an empty inner block element.
```

Comment:
This change is incorrect... it doesn't let you specify the type of element created. When using a string it lets you specify which blocks should be split when the enter key is pressed.

Reply:
We also need to include somewhere that the string value only works in 6.1 or above.